### PR TITLE
Update Secrets dir path

### PR DESCRIPTION
**Pull request by georgeee**

### CODE REVIEW

File: installers/MacInstaller.hs [hunk 1]
```diff
@@ -46,7 +46,7 @@ main = do
   writeFile (dir <> "/Daedalus") $ unlines
     [ "#!/usr/bin/env bash"
     , "cd \"$(dirname $0)\""
-    , "mkdir -p \"$HOME/Library/Application Support/Daedalus/\"{Wallet-0.2,DB-0.2,Logs,Secrets}"
+    , "mkdir -p \"$HOME/Library/Application Support/Daedalus/\"{Secrets-0.5}"
```
Comment (domenkozar) [inline installers/MacInstaller.hs]: Why did you delete the other folders? Fresh install won't work

File: installers/WindowsInstaller.hs [hunk 2]
```diff
@@ -118,8 +118,7 @@ writeInstallerNSIS fullVersion = do
     _ <- section "" [Required] $ do
         setOutPath "$INSTDIR"        -- Where to install files in this section
         writeRegStr HKLM "Software/Daedalus" "Install_Dir" "$INSTDIR" -- Used by launcher batch script
-        createDirectory "$APPDATA\\Daedalus\\Logs"
-        createDirectory "$APPDATA\\Daedalus\\Secrets"
+        createDirectory "$APPDATA\\Daedalus\\Secrets-0.5"
```
Comment (domenkozar) [inline installers/WindowsInstaller.hs]: Why are Logs missing?

File: installers/MacInstaller.hs (hunk 1 above)
Comment (georgeee) [inline installers/MacInstaller.hs]: @domenkozar it will work. Logs dir is created automatically if not exists. Same for databases

File: installers/WindowsInstaller.hs (hunk 2 above)
Comment (georgeee) [inline installers/WindowsInstaller.hs]: Because app will create them automatically

**Comment by georgeee** [comment]
Also I would add that probably app could create this Secrets dir automatically, as it's case for other dirs. We just didn't look at that

File: installers/MacInstaller.hs (hunk 1 above)
Comment (dcoutts) [inline installers/MacInstaller.hs]: Lesson for future. A bash "gotcha" apparently is that a singleton list of {}-expansions does not get expanded, it turns into literal {}s.

    $ echo foo{bar}
    foo{bar}
    $ echo foo{bar,baz}
    foobar foobaz
